### PR TITLE
Fix RTD links to additional tools in the README

### PR DESCRIPTION
**Pull request by bnjmnp**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|    | :bug: Bug fix          |
|    | :sparkles: New feature |
|    | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Just a quick fix of the README

### CODE REVIEW

**Review by DanielNoord — APPROVED** [review]

**Comment by Pierre-Sassoulas** [comment]
Should add redirects, maybe ?

**Comment by Pierre-Sassoulas** [comment]
Thank you for catching this and fixing it @bnjmnp. Do you mind adding a redirect here too please ?

https://github.com/pylint-dev/pylint/blob/614f80e330e30033dd1614187d88b06aa98a1ff3/doc/conf.py#L60-L98

**Comment by Pierre-Sassoulas** [comment]
Yeam seems sensible. It's possible to test on the readthedoc instance generated from the PR : for example : https://pylint--10079.org.readthedocs.build/en/10079/how_tos/index.html redirect as expected). 

In this case:
https://pylint--10079.org.readthedocs.build/en/10079/pyreverse.html
https://pylint--10079.org.readthedocs.build/en/10079/symilar.html
Should work.

**Comment by bnjmnp** [comment]
No luck with my new commit, the links still don't work.
Maybe `../` needs to be removed for the target URLs?

File: doc/conf.py [hunk 1]
```diff
@@ -95,6 +95,8 @@
     "user_guide/output": "usage/output.html",
     "user_guide/pre-commit-integration": "installation/pre-commit-integration.html",
     "user_guide/run": "usage/run.html",
+    "pyreverse": "../additional_tools/pyreverse/index.html",
+    "symilar": "../additional_tools/symilar/index.html",
```
Comment (Pierre-Sassoulas) [inline doc/conf.py:99]:
```suggestion
    "pyreverse": "additional_tools/pyreverse/index.html",
    "symilar": "additional_tools/symilar/index.html",
```

**Review by Pierre-Sassoulas — APPROVED** [review]
Thank you @bnjmnp !

**Comment by bnjmnp** [comment]
It was my pleasure.
Thank you for your work on pylint.

**Pull request merged**